### PR TITLE
popular team and team members

### PR DESCRIPTION
**Pull request by timgcy-snsoft**
will add on members details page after solving all conflicts

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
   },
   "dependencies": {
     "@babel/polyfill": "^7.7.0",
+    "axios": "^0.19.2",
```
Comment (yuhosns) [inline package.json]: you guy was using fetch api, so axios is not necessary here

Comment (timgcy-snsoft) [reply package.json]: I'm using axios at the beginning so it is installed into dependencies, but then i change my method, but did not uninstalled it.

Comment (yuhosns) [reply package.json]: alright, please delete it if not using

File: src/pages/DotaAPI/DotaLandingPage.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import DotaTeam from "../Timothy/proteams";
+export default class DotaApi extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <BasePage title="Dota 2">
+        <BasePage title="Match (KY Chok)">
```
Comment (yuhosns) [inline src/pages/DotaAPI/DotaLandingPage.jsx]: what you need here is heading, basePage is use to act as a wrapper in each page

Comment (timgcy-snsoft) [reply src/pages/DotaAPI/DotaLandingPage.jsx]: ky resolved this on his pc and has pushed

**Pull request merged**